src/pages/Lend/FilterAlpha/LendFilterAlpha.vue

```diff
@@ -91,38 +128,100 @@ export default {
 		KvGrid,
 		KvPageContainer,
 		KvMaterialIcon,
-		LoanCardController
-
+		LoanCardController,
+		KvRadio,
+		KvButton,
 	},
 	data() {
 		return {
 			headerTheme: lightHeader,
 			loanId: Number(this.$route.params.id || 0),
-			// flss query is defaulted to US loans currently to keep loans count
-			// manageable for now
-			loanQueryFilters: { countryIsoCode: { any: ['US'] } },
+			loanQueryFilters: {},
 			totalCount: 0,
 			loans: [],
 			zeroLoans: false,
 			mdiFilterVariant,
 			mdiCompassRose,
+			gender: 'both',
+			sector: ['education', 'agriculture'],
+			country: ['TZ', 'KE'],
 		};
 	},
 	methods: {
+		filterGender() {
+			let genderFilter = {};
+			if (this.gender === 'both') {
+				genderFilter = { eq: {} };
+			} else {
+				genderFilter = { eq: this.gender };
+			}
+			console.log('from genderFilter func:', genderFilter);
+			return genderFilter;
+		},
+		filterSector() {
+			// # TODO: collect sector from checkbox inputs
+			// let sectorFilter = [];
+			// this.sector = ['education', 'agriculture'];
+			const sectorFilter = { any: this.sector };
+			console.log('from filterSector', sectorFilter);
+			return sectorFilter;
+		},
+		filterCountry() {
+			// # TODO: collect country from checkbox inputs
+			// let countryFilter = ['TZ', 'KE'];
+			const countryFilter = { any: this.country };
+			console.log('from filterCountrey', countryFilter);
+			return countryFilter;
+		},
+		resetFilter() {
+			this.loanQueryFilters = {};
+			this.runQuery(this.loanQueryFilters);
+		},
 		runQuery() {
+			console.log('filters into runQuery:', this.loanQueryFilters);
 			fetchData(this.loanQueryFilters, this.apollo).then(flssData => {
 				this.loans = flssData.values ?? [];
 				this.totalCount = flssData.totalCount;
+				console.log('num loans:', this.totalCount);
 
 				if (this.totalCount === 0) {
 					this.zeroLoans = true;
 				}
 			});
 		},
-
+		searchQuery() {
+			this.loanQueryFilters = this.queryFilters;
+			console.log('from searchQuery', this.loanQueryFilters);
+			console.log('new query ran, yes!');
+			this.runQuery();
+		},
```

Comment:
@emuvente  here's the full code. 
on line 193, i'm trying to update the loanQueryFilters
and the line 196, i'm trying to run the graphql query with the filters, I'm getting null loans from FLSS at this point. 
My current train of thought was that this.queryFilters was setting this.loanQueryFilters to the whole Observer object instead of the values?

Reply:
gender component not available in prod yet 🤦